pkg/sloop/store/typed/eventcounttablegen.go

```diff
@@ -331,3 +333,31 @@ func ResourceEventCounts_KeyPredicateFns(keyFn ...func(string) bool) func(string
 		return true
 	}
 }
+
+// Return all keys in all partitions in the given a lookback period
+func (t *ResourceEventCountsTable) GetAllKeysInAllPartitions(db badgerwrap.DB, key *EventCountKey, maxNumberOfKeys int, lookBack int, keyPrefix string) []string {
```

Comment:
Please rename to GetAllKeysForGivenPartitions

Reply:
Fixed!